### ISSUE TITLE
Not able to run the mini app

### ISSUE THREAD

**Issue by minsoe** [electrode-io/MovieListMiniApp#5]
I am not able to run the app. 

```
✖ XCode xcbuild command failed with exit code 65
Error: XCode xcbuild command failed with exit code 65
```
I suspect it is because ElectroeContainer.xcodeproj is not found in the ErnRunner xcode project.
I am not able to find ElectroeContainer.xcodeproj anywhere. 

I've also tried getting start guide but stuck at the **Adding the MoviesApi to the MovieList MiniApp**

Any idea why?

**Comment by minsoe** [comment]
Yup. found it. the path is "/Users/blemair/.ern/containergen/out/ios/ElectrodeContainer.xcodeproj"; instead of "~/.ern/containergen/out/ios/ElectrodeContainer.xcodeproj"

**Comment by belemaire** [comment]
@minsoe 

Good catch.

This is a bug in our runner project generator. 

Opened https://github.com/electrode-io/electrode-native/issues/218
Fix will be included in electrode native 0.9.0 released next Monday.

Meanwhile I will modify the path manually in this MovieListMiniApp pbxproj.

Thanks !
